volmaster/volmaster/cli.go

```diff
@@ -12,6 +12,9 @@ import (
 	"github.com/codegangsta/cli"
 )
 
+// version is provided by build
+var version = ""
```

Comment:
@erikh , this PR will bring in the equivalent of  `docker --version`, which is much smaller and local in scope but is useful for tying our binaries to respective releases.

 `docker version` and `netctl version` have a larger and distributed scope of having a REST endpoint to deal with various distributed components of a system. We should perhaps debate it's need in a separate issue.